query.py

```diff
@@ -88,11 +110,51 @@ def __str__(self):
 class QueryAnd(Query):
     def __init__(self, ops):
         super().__init__()
```

Comment:
We could remove **self.ops_size** field.


Before line 142, 150, 155, set the value for self.total_size.
`self.total_size=len(list(merged))`

In get_size() method, replace with a single line of `return self.total_size`